linked_list/linked_list.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python3
```

Comment:
can you add a Readme file inside the same directory, explaining how the code works. Or links from where other contributors can learn.

Reply:
Sure, I'll add a readme with explanation and some helpful links.